src/lib/java/de/teamlapen/lib/lib/util/UtilLib.java

```diff
@@ -721,4 +724,13 @@ public static AxisAlignedBB MBtoAABB(@Nonnull MutableBoundingBox bb) {
         HUNDRED_EIGHTY,
         TWO_HUNDRED_SEVENTY
     }
+
+    @Nullable
+    public static DyeColor getColorForItem(Item item) {
+        if (!Tags.Items.DYES.contains(item)) return null;
```

Comment:
Not sure if this is helpful for performance because it goes through to the entire set. The following line would go through 16 sets, but with about 1/16 of elements, so it shouldn't be much slower. Don't know, but probably does not matter anyway.